pkg/evetestkit/utils.go

```diff
@@ -663,7 +666,7 @@ func GetRandomAppName(prefix string) string {
 func InitilizeTest(projectName string, options ...TestOption) (*EveNode, error) {
```

Comment:
Can you fix this one too @uncleDecart ?

Reply:
Gosh, weird that you couldn't see my local changes that I haven't pushed :D Now it's fixed

Reply:
Somehow, my AI vision didn't pick up on that. 🤷‍♂️